client/ayon_maya/plugins/load/load_anim.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+import maya.cmds as cmds
+import ayon_api
+from ayon_maya.api import plugin
+from ayon_core.pipeline.load.utils import get_representation_context
+from ayon_maya.plugins.load.load_reference import ReferenceLoader
+import logging
+import pymel.core as pm
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class AnimLoader(plugin.Loader):
+    """Load anim on character"""
+
+    product_types = {"animation"}
+    representations = {"anim"}
+
+    label = "Load Anim"
+    order = -5
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name, namespace, data):
+        project_name = context['project']['name']
+        anim_file = context['representation']['attrib']['path']
+        self.log.info(f"anim_file: {anim_file}")
+        name_space = context['representation']['data']['context']['product']['name'].replace(
+            context['representation']['data']['context']['product']['type'], '')
+        if not cmds.namespace(exists=name_space):
+            assets = context['version']['data'].get("assets", [])
+            self.log.info(f"assets: {assets}")
+            current_asset = [asset for asset in assets if asset['asset_name'] in anim_file]
+            if not current_asset:
+                self.log.warning(f"Asset not found in version data.")
+                return
+            current_asset = current_asset[0]
+            asset_data = ayon_api.get_folder_by_name(project_name=project_name, folder_name=current_asset['asset_name'])
+            versions = ayon_api.get_last_version_by_product_name(project_name, "rigMain", asset_data['id'])
+            representations = ayon_api.get_representations(
+                project_name=project_name,
+                version_ids={versions['id']},
+                fields={"id", "name", "files.path"}
+            )
+            rep_id = None
+            for rep in representations:
+                if rep['name'] == 'ma':
+                    rep_id = rep['id']
+                    break
+            context = get_representation_context(project_name, rep_id)
+            data['attach_to_root'] = True
+            _plugin = ReferenceLoader()
+            _plugin.load(context=context, name=context['product']['name'],
+                                      namespace=name_space, options=data)
+        ctrl_set = pm.ls(name_space + ":rigMain_controls_SET")
```

Comment:
```suggestion
        ctrl_set = pm.ls(name_space + ":rigMain_controls_SET")
```
Keep in mind there is the loaded instance with different variant, e.g. rigTest_controls_SET, rigTechCarA_controls_SET